package.json

```diff
@@ -27,10 +27,13 @@
     "rollup": "^0.26.3"
   },
   "dependencies": {
-    "node-sass": "^3.4.2",
+    "ava": "^0.16.0",
+    "babel-preset-es2015-rollup": "^1.2.0",
+    "node-sass": "^3.8.0",
     "rollup-pluginutils": "^1.3.1"
   },
   "devDependencies": {
+    "ava": "^0.16.0",
```

Comment:
`babel-preset-es2015-rollup` is already in `devDependencies`, and `ava` is added to *both* devDeps and deps in this PR.